playwright.config.ts

```diff
@@ -27,7 +27,7 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 1 : undefined,
```

Comment:
I have only set below env vars in CI now. So `workers` is `undefined`, which means `playwright` manages it automatically. We can always modify this config with a PR.
```
MODE = replay
PORT = 3000
```

Reply:
Is it a big change for that? If not can we do it in this PR itself?

Reply:
Not a big change. I think the situation is we only have few tests to run. The workflow always finishes in less than 10 seconds. Hence not sure what to set.

Reply:
hmm maybe we can have 1 regardless in that case?

Reply:
All 3 sample tests run simultaneously on CI. Should be safe to set 3?

Reply:
Add `IS_CI` env var now.
```
workers: process.env.IS_CI ? 3 : undefined,
reporter: process.env.IS_CI ? 'line' : [ ['html', { outputFolder: './packages/payment-integration-test-framework/report' }] ],
```